.github/workflows/ci.yml

```diff
@@ -37,6 +37,38 @@ jobs:
           file: coverage.xml
           base-path: janus_core
 
+  test-dev-versions:
```

Comment:
I will do this as a separate ci-dev.yml, and also make it really run only on the main repo no need to bother the forks with it.